dashboard/app/models/lesson.rb

```diff
@@ -475,7 +475,7 @@ def summarize_for_rollup(user)
       resources: resources_for_lesson_plan(user&.verified_instructor?),
       vocabularies: vocabularies.sort_by(&:word).map(&:summarize_for_lesson_show),
       programmingExpressions: programming_expressions.sort_by {|pe| pe.syntax || ''}.map(&:summarize_for_lesson_show),
-      objectives: objectives.sort_by(&:description).map(&:summarize_for_lesson_show),
+      objectives: objectives.sort_by(&:updated_at).map(&:summarize_for_lesson_show),
```

Comment:
It is possible that this change isn't wanted/needed.  This is for `summarize_for_rollup` but I don't know how TNL wants the roll up view to be shown...